stl/inc/__msvc_format_ucd_tables.hpp

```diff
@@ -124,7 +124,6 @@ struct _Unicode_property_data {
 // Codepoint ranges may not overlap, and, within one property, a codepoint may only appear once. Furthermore the
 // codepoint lower bounds appear in sorted (ascending) order.
 
-

```

Comment:
From this file:

https://github.com/microsoft/STL/blob/16bb556afe5c56fe70a5508a6116842c226dd3be/stl/inc/__msvc_format_ucd_tables.hpp#L6-L8

I think the correct way to remove those whitespaces would be modifying `grapheme_break_property_data_gen.py` by calling `lstrip` before `format` here:

https://github.com/microsoft/STL/blob/16bb556afe5c56fe70a5508a6116842c226dd3be/tools/unicode_properties_parse/grapheme_break_property_data_gen.py#L231-L232

Reply:
Good catch, I will revise.